sqlmesh/core/state_sync/base.py

```diff
@@ -190,23 +191,23 @@ def raise_error(
                 f"{lib} (local) is using version '{local}' which is behind '{remote}' (remote).{upgrade_suggestion}"
             )
 
-        if SCHEMA_VERSION < versions.schema_version:
+        if SCHEMA_VERSION < versions.schema_version and not force:
```

Comment:
I think we can simplify this slightly and instead of adding `not force` to every branch we can just add the following at line 171:
```
if not force:
    return versions
```
to circumvent all validations. 

Alternatively, @tobymao @z3z1ma  , shouldn't `validate=False` take care of this?

Reply:
yea i'm a bit confused about why this new flag was needed.

Reply:
@z3z1ma let's remove the `force` flag and immediately return `versions if`validate=False`